tests/unit_tests/test_wave_06.py

```diff
@@ -261,7 +281,18 @@ def test_swap_best_by_category_no_other_match_is_false():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    tai_list = [item_a, item_b, item_c]
+    jesse_list = [item_d, item_e, item_f]
+
+    assert not result
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert all (item in tai.inventory for item in tai_list)
+    assert all (item in jesse.inventory for item in jesse_list)
```

Comment:
👀 Here as well, you can either implement your own version of `all` and use it here, or use an approach that doesn't need `all`.